// @starknet-react/core/dist/network.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.chainById = void 0;
const starknet_1 = require("starknet");
/** Returns a chain information from its id. */
function chainById(chainId) {
    return KNOWN_CHAINS[chainId];
}
exports.chainById = chainById;
const KNOWN_CHAINS = {
    [starknet_1.constants.StarknetChainId.MAINNET]: {
        id: starknet_1.constants.StarknetChainId.MAINNET,
        name: 'StarkNet Mainnet',
        blockExplorer: {
            name: 'Voyager',
            url: 'https://voyager.online',
        },
        testnet: false,
    },
    [starknet_1.constants.StarknetChainId.TESTNET]: {
        id: starknet_1.constants.StarknetChainId.TESTNET,
        name: 'StarkNet Görli',
        blockExplorer: {
            name: 'Voyager',
            url: 'https://goerli.voyager.online',
        },
    },
    [starknet_1.constants.StarknetChainId.TESTNET2]: {
        id: starknet_1.constants.StarknetChainId.TESTNET2,
        name: 'StarkNet Görli 2',
        blockExplorer: {
            name: 'Voyager',
            url: 'https://goerli-2.voyager.online',
        },
    },
};
